src/streamlink_cli/utils/player.py

```diff
@@ -15,23 +15,23 @@ def check_paths(exes, paths):
 def find_default_player():
     if "darwin" in sys.platform:
         paths = os.environ.get("PATH", "").split(":")
-        paths += ["/Applications/VLC.app/Contents/MacOS/"]
-        paths += ["~/Applications/VLC.app/Contents/MacOS/"]
-        path = check_paths(("VLC", "vlc"), paths)
+        paths += ["/Applications/mpv.app/Contents/MacOS/"]
+        paths += ["~/Applications/mpv.app/Contents/MacOS/"]
+        path = check_paths(("MPV", "mpv"), paths)
     elif "win32" in sys.platform:
-        exename = "vlc.exe"
+        exename = "mpv.exe"
         paths = os.environ.get("PATH", "").split(";")
         path = check_paths((exename,), paths)
 
         if not path:
-            subpath = "VideoLAN\\VLC\\"
+            subpath = "mpv-x86_64\\"
             envvars = ("PROGRAMFILES", "PROGRAMFILES(X86)", "PROGRAMW6432")
             paths = filter(None, (os.environ.get(var) for var in envvars))
             paths = (os.path.join(p, subpath) for p in paths)
             path = check_paths((exename,), paths)
     else:
         paths = os.environ.get("PATH", "").split(":")
-        path = check_paths(("vlc",), paths)
+        path = check_paths(("mpv",), paths)
```

Comment:
maybe this could be used for vlc and mpv,
so the behavior for a default vlc don't change.

example for _linux_

```py
path = check_paths(("vlc",), paths)
if not path:
    path = check_paths(("mpv",), paths)
```

Reply:
I think having a list of possible players by default would be advantageous to the user. If they have a specific player then they set it in the config, same as it ever was. 

I would say that prioritising mpv would be a good idea too, but that should probably done in a major version bump.